File: plugins/OrchidPluginDocs/src/main/java/com/eden/orchid/pluginDocs/PluginDocsResourceSource.java
package com.eden.orchid.pluginDocs;

import com.eden.orchid.api.OrchidContext;
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource;

import javax.inject.Inject;

public class PluginDocsResourceSource extends PluginResourceSource {

    @Inject
    public PluginDocsResourceSource(OrchidContext context) {
        super(context, 1000);
    }

}

File: plugins/OrchidPluginDocs/src/main/kotlin/com/eden/orchid/plugindocs/PluginDocsResourceSource.kt
package com.eden.orchid.plugindocs

import com.eden.orchid.api.OrchidContext
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource

import javax.inject.Inject

class PluginDocsResourceSource @Inject
constructor(context: OrchidContext) : PluginResourceSource(context, 1000)
